.github/e2e-tests-olm/action.yaml

```diff
@@ -75,6 +79,46 @@ runs:
           --namespace operators \
           --skip-tls
 
+        # NOTE: operator will deploy other pods as well which we don't want to wait for
+        kubectl wait --for=condition=Ready pods \
```

Comment:
This looks a bit fragile tbh. If we change the labels, it might be hard to find why e2e tests suddenly start to fail. I would suggest that we simply wait for all pods in the `operators` namespace

Reply:
I did think of all pods but then it will also wait for pods that operator (controllers in operator) starts.

Reply:
This should be solved in theory when your sig-instrumentation patch merges upstream and we should be able to use promq as dependency.

Reply:
Can we not wait only for the operator to start? E.g.
```
kubectl rollout status -n operators deploy monitoring-stack-operator
```

Reply:
Yes, that is a way better idea!